From 3f4907c5c65c782e3ab03aa965d3fd558a986ef2 Mon Sep 17 00:00:00 2001
From: Benjamin Berg <bberg@redhat.com>
Date: Wed, 9 Oct 2019 22:41:27 +0200
Subject: [PATCH 2/2] data: Make X11 services part of gnome-session

The X11 services need to be stopped together with the session. Add
corresponding Requisite/PartOf/Before entries.
---
 data/gnome-session-x11-services.target | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/gnome-session-x11-services.target b/data/gnome-session-x11-services.target
index e9dea9b5..d8d2883d 100644
--- a/data/gnome-session-x11-services.target
+++ b/data/gnome-session-x11-services.target
@@ -1,3 +1,7 @@
 [Unit]
 Description=GNOME session X11 services
 DefaultDependencies=no
+
+Requisite=gnome-session.target
+PartOf=gnome-session.target
+Before=gnome-session.target
-- 
2.23.0

